### PR TITLE
Corrections to API input checking

### PR DESCRIPTION
**Pull request by dcolley**
## Description

Minor typos in api.py for input validation

## Type of change

Please make your selection.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a new dependency
- [ ] Code Maintainability / comments
- [ ] Code Refactoring / future-proofing

## How Has This Been Tested?

N/A

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added pytest unit tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: models/exchange/coinbase_pro/api.py [hunk 1]
```diff
@@ -320,7 +320,7 @@ def limitSell(self, market: str='', base_quantity: float=0, futurePrice: float=0
         if not isinstance(base_quantity, int) and not isinstance(base_quantity, float):
             raise TypeError('The crypto amount is not numeric.')
 
-        if not isinstance(base_quantity, int) and not isinstance(base_quantity, float):
+        if not isinstance(futurePrice, int) and not isinstance(futurePrice, float):
```
Comment (whittlem) [inline models/exchange/coinbase_pro/api.py:323]: Can you change "futurePrice" too "future_price" please? Thanks.

Comment (whittlem) [reply models/exchange/coinbase_pro/api.py:323]: @dcolley, are you going to make this change?

Comment (dcolley) [reply models/exchange/coinbase_pro/api.py:323]: yes, I'll make the change and revert

Comment (whittlem) [reply models/exchange/coinbase_pro/api.py:323]: @dcolley, thanks. I'll merge it when is see your update.

**Pull request merged**